### DISCUSSION TITLE
Error messages for build.gradle in 1.25.0

### DISCUSSION THREAD

**Discussion by treevis** [SchoofsKelvin/vscode-sshfs#343] [Q&A] [answered]
After updating to 1.25.0, I now see two instances of the following error when I choose "Add as Workspace folder" from my SSH FS configuration. This error does not present itself when I switch back to 1.24.1 or if I choose "Open remote SSH terminal".

```
Error handling uri:
ssh://summer/<path/to/my/folder>/build.gradle
ssh://summer/<path/to/my/folder>/build.gradle
```

![build_gradle](https://user-images.githubusercontent.com/645640/172412855-c36ae56a-4e78-4793-a5ab-b84f6f9457f2.png)

SSH FS Configuration:
```
"sshfs.configs": [
    {
        "name": "summer",
        "host": true,
        "root": "/<path/to/my/folder>",
        "agent": "$SSH_AUTH_SOCK",
        "username": "$USERNAME"
    }
]
```

When I enabled debugging, I see that SSH FS looks for a few files in the folder and writes an error or warning when it can't find it. This happens in 1.24.1 and 1.25.0. Only 1.25.0 reports out this `build.gradle` file in the app. It doesn't report the others files that it doesn't find on the server. Do I need to change something in my configuration so this error doesn't show up every time I connect?

**Comment by SchoofsKelvin** [comment, marked as answer]
Looks like you're also suffering from #341? You can try the build I linked in there (or just the [latest build](https://github.com/SchoofsKelvin/vscode-sshfs/actions/workflows/build-extension.yml?query=branch%3Amaster) of the `master` branch), although I might end up releasing a new version of the extension tonight, as I imagine it might affect several people.

Small recap from what's been spoken about in that issue: some extension (Maven/Gradle extension?) or built-in code tries to directly read `build.gradle`, without even checking whether it exists. Version v1.25.0 added a notification for nearly all failed file system operations due to VS Code somehow _not_ informing the user about e.g. failed save operations since (if I remember correctly) VS Code 1.56.0. The `master` branch now has some improvements to fine-tune these errors to be 1) only shown for failed write operations and 2) configurable with flags instead of being hardcoded enabled.

**Reply by treevis** [reply]
Yes, #341 is the same issue as I am seeing - I missed that when I was searching the issues. Yes, the latest build you linked to addresses the issue for me.

Thanks for the help.
